repo/packages/M/minio/10/marathon.json.mustache

```diff
@@ -0,0 +1,86 @@
+{
+   "id":"{{service.name}}",
+   "cpus":{{resource.cpus}},
+   "mem":{{resource.mem}},
+   "instances":1,
+   "maxLaunchDelaySeconds":36000,
+   "args":[
+      "server",
+      "-C",
+      "/config",
+      "/export"
+   ],
+   "container":{
+      "type":"DOCKER",
+      "volumes": [
+        {
+            "containerPath": "miniodata",
+            "mode": "RW",
+            "persistent": {
+            "type": "root",
+            "size" : {{storage.volume-size}}
+            }
+        },
+        {
+            "containerPath": "minioconfig",
+            "mode": "RW",
+            "persistent": {
+            "type": "root",
+            "size": 10
+            }
+        },
+        {
+            "containerPath": "/export",
+            "hostPath": "miniodata",
+            "mode": "RW"
+        },
+        {
+            "containerPath": "/config",
+            "hostPath": "minioconfig",
+            "mode": "RW"
+        }
+      ],
+      "docker":{
+         "image": "{{resource.assets.container.docker.minio-docker-RELEASE}}",
+         "network":"BRIDGE",
+         "portMappings":[
+            {
+               "containerPort":9000,
+               "hostPort":0,
+               "servicePort":{{networking.port}}
+            }
+         ],
+         "privileged":true,
+         "forcePullImage":true
+      }
+   },
+   "healthChecks": [
```

Comment:
Reposting what @joerg84 mentioned on slack. 

https://dcos.io/docs/1.10/deploying-services/creating-services/health-checks/ 
```
{
  "protocol": "COMMAND",
  "command": { "value": "source ./myHealthCheck.sh" }
}
```

We should re-do our healthchecks to use this style.

Reply:
I ran into that issue with the existing package, if the above hc works I am fine with that.